qiskit_nature/problems/sampling/sampling_problem.py

```diff
@@ -0,0 +1,27 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An interface for sampling problems."""
+from abc import ABC, abstractmethod
+
+
+class SamplingProblem(ABC):
+    """An interface for sampling problems."""
+
+    @abstractmethod
+    def qubit_op(self):
```

Comment:
Can we put return typehints on the functions and have the returns as part of normal Returns: aspect of docstring